### ISSUE TITLE
Not in Package Control?

### ISSUE THREAD

**Issue by hdavidzhu** [shime/sublime-livedown#4]
Currently, there's no way to install this through Package Control. How does that get set up?

**Comment by shime** [comment]
installation instructions are in the [readme](https://github.com/shime/sublime-livedown#installation)

**Comment by hdavidzhu** [comment]
Yep, that's how I installed it. Just wondering if you'd ever move it to Package Control as well. Thanks!

**Comment by shime** [comment]
thanks for bringing it up, I'll try to find some time for that soon.

**Comment by hdavidzhu** [comment]
Cool! BTW this is an amazing plugin :D

**Comment by shime** [comment]
thanks! :+1: glad to hear it's helpful!
